backend/typescript/graphql/resolvers/entityResolvers.ts

```diff
@@ -8,6 +8,7 @@ import {
   validateFileType,
   getFileTypeValidationError,
 } from "../../middlewares/validators/util";
+import { generateCSV } from "../../utilities/CSVUtils";
```

Comment:
Can you please revert all the TS changes? Will merge the TS implementation with #91.